tutor/src/components/tours/conductor.jsx

```diff
@@ -40,7 +44,13 @@ class TourConductor extends React.Component {
     }
   }
 
+  @computed get ready() {
+    return this.tourContext.ready;
+  }
+
```

Comment:
same as other spot, this probably should be `isReady`, although I see tourContext doesn't follow that "rule" either.